web/src/engine/websites/ReaperScans.ts

```diff
@@ -1,42 +1,35 @@
 import { Tags } from '../Tags';
 import icon from './ReaperScans.webp';
-import { type MangaPlugin, DecoratableMangaScraper, type Manga, Chapter, type Page } from '../providers/MangaPlugin';
+import { type MangaPlugin, DecoratableMangaScraper, Manga, Chapter, type Page } from '../providers/MangaPlugin';
 import * as Common from './decorators/Common';
-import { FetchCSS, FetchJSON } from '../platform/FetchProvider';
 import { Priority, TaskPool } from '../taskpool/TaskPool';
 import { RateLimit } from '../taskpool/RateLimit';
 import { Numeric } from '../SettingsManager';
 import { WebsiteResourceKey as R } from '../../i18n/ILocale';
+import { FetchJSON } from '../platform/FetchProvider';
 
-type LaravelLivewireMessage = {
-    effects?: {
-        html: string;
-    };
-    fingerprint: {
-        name: string;
-    };
-    serverMemo: {
-        data: {
-            page: number;
-            paginators: {
-                page: number;
-            }
-        }
+type APIResult<T> ={
+    data : T
+}
+
+type APIManga = {
+    id: number
+    title: string,
+}
+
+type APIChapter = {
+    chapter_slug: string,
+    chapter_name: string
+    series: {
+        series_slug : string
     }
-    updates?: {
-        type: string;
-        payload: {
-            id: string;
-            method: string;
-            params: Array<number|string>;
-        }
-    }[];
 }
 
 @Common.ImageAjax()
 export default class extends DecoratableMangaScraper {
 
     private readonly interactionTaskPool = new TaskPool(1, RateLimit.PerMinute(15));
```

Comment:
Is the endpoint still rate limited?

Reply:
To be frank i havent tested but id rather avoid bad surprises. Its Reaper scans after all.